### PR TITLE
Internal: Add class attribute to Atomic Image

### PR DESCRIPTION
**Pull request by yotamselementor**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/atomic-widgets/widgets/atomic-image.php [hunk 1]
```diff
@@ -29,13 +30,19 @@ public function get_name() {
 	protected function render() {
 		$settings = $this->get_atomic_settings();
 
-		$image_url = $settings['image'];
+		if ( empty( $settings['image'] ) ) {
+			return;
```
Comment (IshayMaya) [inline modules/atomic-widgets/widgets/atomic-image.php:34]: @yotamselementor not sure it's related to this PR but as i understood we will not support having empty url right? should always have some placeholder

File: modules/atomic-widgets/widgets/atomic-image.php [hunk 2]
```diff
@@ -29,13 +30,19 @@ public function get_name() {
...
+		}
+
+		$image_url = esc_url( $settings['image'] );
+		$src = 'src=' . $image_url;
+		$class = '';
```
Comment (IshayMaya) [inline modules/atomic-widgets/widgets/atomic-image.php:39]:
```suggestion
		$class = '';
		
```

File: modules/atomic-widgets/widgets/atomic-image.php (hunk 1 above)
Comment (yotamselementor) [reply modules/atomic-widgets/widgets/atomic-image.php]: Good point, fixed: https://github.com/elementor/elementor/pull/28626/commits/ce528555002965302bd918842c725c9e258ef692

File: modules/atomic-widgets/widgets/atomic-image.php (hunk 2 above)
Comment (yotamselementor) [reply modules/atomic-widgets/widgets/atomic-image.php:35]: Fixed: https://github.com/elementor/elementor/pull/28626/commits/96329caf29a386f26289378fcedd30678842b6d3

**Review by IshayMaya — APPROVED** [review]

**Pull request merged**